crates/swc_ecma_transforms_compat/tests/es2020_optional_chaining.rs

```diff
@@ -303,3 +303,16 @@ fn fixture_loose(input: PathBuf) {
         Default::default(),
     );
 }
+
+#[testing::fixture("tests/optional-chaining-private-var/issue-7561/input.js")]
```

Comment:
Please remove this

Reply:
You don't need to add a new function with `testing::fixture`

Reply:
Just add a test file to the correct directory.

Reply:
I think I've added it to the correct folder.